### DISCUSSION TITLE
希望这款插件能够更加易用

### DISCUSSION THREAD

**Discussion by fangjia052** [IrineSistiana/mosdns#118] [Ideas]
希望可以做成像SmartDNS一样可以直接在OpenWrt的管理界面中进行配置和使用，便于大众化使用，这样也有利于更多人知道和了解这款插件，将这款插件逐渐完善~

**Comment by xiaoyeziwudi** [comment]
可能作者并不想让大众用户使用。说实在话,我装了一个debian的虚拟机，都不知道怎么安装。wget下载一个zip的压缩包，然后解压到文件夹，就不知道怎么运行了。
